shared/oval/disable_interactive_boot.xml

```diff
@@ -3,7 +3,7 @@
     <metadata>
       <title>Disable Interactive Boot</title>
       <affected family="unix">
-        <platform>multi_platform_rhel</platform>
+        <platform>Red Hat Enterprise Linux 7</platform>
```

Comment:
I am stumbling up-on this.

This is in `shared/` directory. And the file seems to be used by RHEL6 profiles.

This is just a minor observation, as the `<platform>` element is not interpreted by scanners. Still the todo bellow perhaps applies also only to RHEL7 part.